recipes-support/salt/salt_3000.2.bb

```diff
@@ -0,0 +1,203 @@
+HOMEPAGE = "http://saltstack.com/"
+SECTION = "admin"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=c996f5a78d858a52c894fa3f4bec68c1"
+DEPENDS = "\
+           python3-msgpack \
+           python3-pyyaml \
+           python3-jinja2 \
+           python3-markupsafe \
+"
+
+DEPENDS = "\
+           python3-distro-native \
+"
+
+PACKAGECONFIG = "tcp"
+PACKAGECONFIG[tcp] = ",,python3-pycrypto"
+
+SRC_URI = "git://github.com/ni/salt.git;protocol=https;branch=ni/master/3000.2 \
+           file://set_python_location_hashbang.patch \
+           file://minion \
+           file://salt-minion \
+           file://salt-common.bash_completion \
+           file://salt-common.logrotate \
+           file://salt-api \
+           file://salt-master \
+           file://master \
+           file://salt-syndic \
+           file://cloud \
+           file://roster \
+"
+
+SRCREV = "${AUTOREV}"
+
+S="${WORKDIR}/git"
+
+inherit setuptools3 update-rc.d
+
+# Avoid a QA Warning triggered by the test package including a file
+# with a .a extension
+INSANE_SKIP_${PN}-tests += "staticdev"
+
+# Note ${PN}-tests must be before ${PN}-common in the PACKAGES variable
+# in order for ${PN}-tests to own the correct FILES.
+PACKAGES += "\
```

Comment:
FYI: out of all those packages we only use (and test) salt-minion and salt-common, so if it helps you could get rid of the others.

Reply:
Ok, I'll see how well I can clean up the remaining packages.

Reply:
I've looked into this some more, and I won't be able to remove the other packages cleanly. The files - salt-syndic, salt-cloud, etc.- currently owned by the other packages are installed by distutils. And if there are no corresponding salt packages in the recipe that claim ownership of these files, bitbake errors out during the build. So I think I will let the remaining packages be as is, instead of trying to remove these files from the installed directories.
@amstewart  thoughts?

Reply:
It will probably be more maintenance burden to remove the packages than to maintain them - since bitbake will normally complain if all of the files installed to the sysroot don't get packaged *somewhere*. So I would recommend you leave them in the recipe.